modules/extensions-spring-support/src/main/java/org/mule/runtime/module/extension/internal/config/dsl/AbstractExtensionObjectFactory.java

```diff
@@ -135,4 +146,37 @@ private String unwrapChildKey(String key)
     {
         return key.replaceAll(CHILD_ELEMENT_KEY_PREFIX, "").replaceAll(CHILD_ELEMENT_KEY_SUFFIX, "");
     }
+
+    private void checkParameterGroupExclusivenessForModel(EnrichableModel model, String component, String componentName, Set<String> resolverKeys) throws ConfigurationException
```

Comment:
String component and String componentName is not a good signature. Strings are evil. Why do you need those? Why can't they be part of the model property itself? At the very least, use better names. But Strings are evil when packed